packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.tsx

```diff
@@ -177,40 +156,71 @@ export type CardContainerLogicProps = {
  * Class wraps around CardContainer and serves as a standalone logic container for rendering cards.
  */
 export function CardContainerLogic(props: CardContainerLogicProps) {
-  const entityId = parseEntityIdFromSqlStatement(props.sql)
-  const queryFilters = getAdditionalFilters(
+  const { sql: deprecatedSql } = props
```

Comment:
Needed to update logic to either 
1. Use `query` prop
2. If no `query` prop, use the deprecated `sql`, `searchParams`, `sqlOperator`